lang/main.json

```diff
@@ -401,26 +410,31 @@
     },
     "recording":
     {
-        "pending": "Recording waiting for a member to join...",
-        "on": "Recording",
-        "off": "Recording stopped",
-        "failedToStart": "Recording failed to start",
+        "busy": "We're working on freeing recording resources. Please try again in a few minutes.",
+        "busyTitle": "All recorders are currently busy",
         "buttonTooltip": "Start / Stop recording",
         "error": "Recording failed. Please try again.",
-        "unavailable": "The recording service is currently unavailable. Please try again later."
+        "failedToStart": "Recording failed to start",
+        "off": "Recording stopped",
+        "on": "Recording",
+        "pending": "Recording waiting for a member to join...",
+        "unavailable": "Oups! The recording service is currently unavailable. We're working on resolving the issue. Please try again later.",
+        "unavailableTitle": "Recording unavailable"
     },
     "liveStreaming":
     {
-        "pending": "Starting Live Stream...",
-        "on": "Live Streaming",
-        "off": "Live Streaming Stopped",
-        "unavailable": "The live streaming service is currently unavailable. Please try again later.",
-        "failedToStart": "Live streaming failed to start",
+        "busy": "We're working on freeing streaming resources. Please try again in a few minutes.",
+        "busyTitle": "All streamers are currently busy",
         "buttonTooltip": "Start / Stop live stream",
+        "error": "Live streaming failed. Please try again.",
+        "failedToStart": "Live streaming failed to start",
+        "off": "Live Streaming Stopped",
+        "on": "Live Streaming",
+        "pending": "Starting Live Stream...",
```

Comment:
While you're in here changing the blame anyway, would you consider (with no obligation to actually change) the capitalization of the live streaming strings. I noticed they're Title Cased but other strings are not.

Reply:
Not sure what you mean actually, could you please explain? While I've seen other strings that were capitalised, the majority of the string keys are following the same rule as those for live streaming.

Reply:
error, failedToStart, unavailable, streamIdRequired, and unavailableTitle treat live stream not as a proper noun and so do not capitalize the first letters of "live stream." However, off, on, and pending capitalize live stream like it's a proper noun.

Reply:
Got it, will fix!